app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewCard.kt

```diff
@@ -0,0 +1,66 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+data class CrewCardCardViewState(
+    val name: String,
+    val role: String,
+)
+
+@Composable
+fun CrewCard(
+    crewCardCardViewState: CrewCardCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Surface(color = MaterialTheme.colors.background,
```

Comment:
formatting - function calls with multiple parameters should have every parameter in the new line, including first parametere

Reply:
Please check all files